composer.json

```diff
@@ -30,5 +30,10 @@
     "autoload": {
         "psr-0": { "Liip\\TranslationBundle": "" }
     },
-    "target-dir": "Liip/TranslationBundle"
+    "target-dir": "Liip/TranslationBundle",
```

Comment:
yeah .. pondered that .. I am not so excited about PSR-4 .. but I am open to PRs :)